newrelic/structures_newrelic_one_dashboard.go

```diff
@@ -169,6 +169,7 @@ func expandDashboardPageInput(d *schema.ResourceData, pages []interface{}, meta
 			page.GUID = common.EntityGUID(guid.(string))
 		}
 
+		page.Widgets = []dashboards.DashboardWidgetInput{}
```

Comment:
Note for the reviewer
- The list has been initialised here in order to be able to send an empty list to the API even when no widgets are added to the Terraform Configuration
- Without this change, an empty list would not be sent to client-go when no widgets are added, and instead, **null** would be sent, causing the existing problem again